tests/test_ctf.py

```diff
@@ -0,0 +1,97 @@
+"""Test function for CTF module."""
+import numpy as np
+import torch
+
+from simSPI.linear_simulator.ctf_utils import CTF
```

Comment:
`ctf_utils` should just be called `ctf.py` (give it the name of the principal object inside it).

Reply:
Done!